README.md

```diff
@@ -118,7 +127,13 @@ Setup instructions as follows:
 * Run test_bulk.py
 
 ### Command set
-The most accurate way to understand the commands is to read [the code](https://github.com/madcowswe/ODriveFirmware/blob/f19f1b78de4bd917284ff95bc61ca616ca9bacc4/MotorControl/low_level.c#L353) that parses the commands.
+The most accurate way to understand the commands is to read [the code](https://github.com/madcowswe/ODriveFirmware/blob/f19f1b78de4bd917284ff95bc61ca616ca9bacc4/MotorControl/vommands.c#L133) that parses the commands. Nevertheless, here is an overview:
```

Comment:
Typo.  vommands.c -> commands.c

Also, this should probably use the branch name and not a random commit.  Correct line: https://github.com/madcowswe/ODriveFirmware/blob/master/MotorControl/commands.c#L133